e3sm_diags/driver/zonal_mean_xy_driver.py

```diff
@@ -1,262 +1,316 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, List, Tuple
 
-import cdms2
-import cdutil
-import MV2
-import numpy
+import xarray as xr
 
-from e3sm_diags.driver import utils
+from e3sm_diags.driver.utils.dataset_xr import Dataset
+from e3sm_diags.driver.utils.regrid import (
+    get_z_axis,
+    has_z_axis,
+    regrid_z_axis_to_plevs,
+)
+from e3sm_diags.driver.utils.type_annotations import MetricsDict
 from e3sm_diags.logger import custom_logger
-from e3sm_diags.metrics import corr, max_cdms, mean, min_cdms, rmse
-from e3sm_diags.plot import plot
+
+# TODO
+# from e3sm_diags.driver.utils.io import _save_data_metrics_and_plots
```

Comment:
This commented line was higher in the import list which caused `isort` to throw an error about a blank line in between imports. Moving it down here fixes that issue for now.

I removed the `.pre-commit-config.yaml` file update for `isort` to use the `black` profile since it isn't needed.